src/xunit.netcore.extensions/xunit.netcore.extensions.csproj

```diff
@@ -10,6 +10,8 @@
     <RootNamespace>Xunit.NetCore.Extensions</RootNamespace>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
```

Comment:
You are correct; i was importing the packages incorrectly. I tried it with both versions a few times to ensure it works either way and will be updating the PR with them removed